.drone.yml

```diff
@@ -0,0 +1,35 @@
+matrix:
+  DOCKER_REPO:
+    - ukhomeofficedigital/helm
+  DOCKER_REGISTRY:
+    - quay.io
+  DOCKER_USERNAME:
+    - ukhomeofficedigital+helm
+  DOCKER_HOST:
+    - tcp://172.17.0.1:2375
+pipeline:
+  docker-build:
+    image: docker:19.03.5
+    commands:
+      - docker build -t ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA} .
+    when:
+      event:
+        - pull_request
+        - push
+        - tag
+
+  docker-push:
+    image: docker:19.03.5
+    environment:
+      docker_username: ukhomeofficedigital+helm
+    secrets:
+      - docker_password
+    commands:
+      - docker login -u="${DOCKER_USERNAME}" -p=$${DOCKER_PASSWORD} ${DOCKER_REGISTRY}
+      - docker tag ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA} ${DOCKER_REGISTRY}/${DOCKER_REPO}:latest
+      - docker tag ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA} ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_TAG}
+      - docker push ${DOCKER_REGISTRY}/${DOCKER_REPO}:$${DRONE_COMMIT_SHA}
+      - docker push ${DOCKER_REGISTRY}/${DOCKER_REPO}:latest
```

Comment:
We may not want to overwrite latest on every tag push

Reply:
Good call. I'll add a pipeline step triggered by a push to that master to deal with it. That should cover it, shouldn't it?

Reply:
Yup that would be good, thank you :)